content/_posts/2020-04-06-template-oriented-programming-talk.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Template-Oriented-Programming to Ship Faster, Part-1
+header-image: https://www.dropbox.com/s/249l72ehauel1np/guanlet-2.gif
```

Comment:
```suggestion
header-image: https://img.youtube.com/vi/_QBlKtUY6ac/maxresdefault.jpg
```

Following `https://img.youtube.com/vi/<ID>/maxresdefault.jpg` pattern so we don't depend on Dropbox accounts :+1:

Reply:
Thanks, Done!